lisa/testsuite.py

```diff
@@ -265,6 +271,32 @@ def check_platform(
 
         return result
 
+    def get_log_file_handler(self) -> logging.FileHandler:
```

Comment:
Have a paired methods called, `subscribe_log` and `unsubscribe_log`, and pass in the logger to be subscribed.